authorization-common/src/main/java/com/sequenceiq/authorization/aspect/CheckPermissionsAspects.java

```diff
@@ -19,9 +19,9 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(CheckPermissionsAspects.class);
 
     @Inject
-    private PermissionCheckerService permissionCheckerService;
+    private AbstractPermissionCheckerService permissionCheckerService;
 
-    @Pointcut("execution(public * com.sequenceiq.cloudbreak.repository..*.*(..)) ")
+    @Pointcut("execution(public * com.sequenceiq..repository..*.*(..)) ")
```

Comment:
As we agreed at your desk, ples make this aspect to look for subclasses of BaseWhateverRepository that we have to extend in our repos anyway, so we won't have package restrictions.

Reply:
(and place back the moved repos)